includes/jetpack-settings/share-image/class-share-image.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Share Image
+ * 
+ * @package Cata\SEO\Jetpack_Settings
+ */
+
+namespace Cata\SEO\Jetpack_Settings;
+
+use WP_Customize_Manager;
+use WP_Customize_Media_Control;
+
+/**
+ * Share Image
+ */
+class Share_Image {
+	/**
+	 * Construct
+	 */
+	public function __construct() {
+		/**
+		 * Add share image setting to customizer
+		 */
+		add_action( 'customize_register', array( __CLASS__, 'add_share_image' ), 10 );
+		add_action( 'customize_register', array( __CLASS__, 'add_customizer_section' ), 10 );
+		add_action( 'customize_register', array( __CLASS__, 'add_share_image_setting' ), 20 );
+
+		/**
+		 * Use share image
+		 */
+		add_filter( 'jetpack_open_graph_tags', array( __CLASS__, 'add_homepage_share_image' ) );
+		add_filter( 'jetpack_twitter_cards_image_default', array( __CLASS__, 'get_homepage_share_image_url' ) );
+	}
+
+	/**
+	 * Add Share Image Panel
+	 *
+	 * @param WP_Customize_Manager $wp_customize - instance of WP_Customize_Manager.
+	 */
+	public static function add_share_image( WP_Customize_Manager $wp_customize ): void {
+		$wp_customize->add_panel(
+			'cata_seo_jetpack_settings_share_image_panel',
+			array(
+				'title'       => __( 'Share Image', 'cata' ),
+				'description' => __( 'Edit Homepage Share Image', 'cata' ),
+				'capability'  => 'edit_theme_options',
+				'priority'    => 10,
+			) 
+		);
+	}
+
+	/**
+	 * Add Customizer Section
+	 * 
+	 * @param WP_Customize_Manager $wp_customize - instance of WP_Customize_Manager.
+	 */
+	public static function add_customizer_section( WP_Customize_Manager $wp_customize ): void {
+		/**
+		 * Add a Customizer Section for Creepy Catalog theme.
+		 */
+		$section_options = array(
+			'title'       => __( 'Homepage' ),
+			'description' => __( 'Settings used by the Creepy Catalog theme.' ),
```

Comment:
Can you generalize this description and name?

Reply:
Yep, I made this less specific